MANIFEST.in

```diff
@@ -1,5 +0,0 @@
-include versioneer.py
```

Comment:
I think our pyproject already has the versioneer infrastructure, so this should not be necessary